templates/AP_app/approve-taxpayer.html

```diff
@@ -0,0 +1,5 @@
+<form method="POST" action="{% url 'change-taxpayer-status' taxpayer_id=taxpayer.id %}">
+    {% csrf_token %}                
+    <input type="hidden" name="action" value="approve">                
+    <input class='btn btn-success' type="submit" value="Approve" id="approve"/>
+</form>
```

Comment:
This could be joined with `deny-taxpager.html`

Reply:
I separate both because I have to use them separately in another template. For example if the AP doesn't set the workday id the template only shows the deny button